src/modules/ticket/reducer.ts

```diff
@@ -0,0 +1,89 @@
+import {Alert} from 'react-native';
+import {Action} from 'redux';
+import {t} from '@translations';
+
+export const BUY_TICKET_REQUEST = 'BUY_TICKET_REQUEST';
+export const BUY_TICKET_SUCCESS = 'BUY_TICKET_SUCCESS';
+export const BUY_TICKET_FAIL = 'BUY_TICKET_FAIL';
+
+export interface TicketState {
+  isLoading: boolean;
+  error: string | null;
+  reservationId: string | null;
+}
+
+export interface BuyTicketRequestAction extends Action {
+  type: typeof BUY_TICKET_REQUEST;
+  payload: string;
+}
+
+export interface BuyTicketSuccessAction extends Action {
+  type: typeof BUY_TICKET_SUCCESS;
+  payload: {
+    reservationId: string;
+  };
+}
+
+export interface BuyTicketFailAction extends Action {
+  type: typeof BUY_TICKET_FAIL;
+  payload: {
+    error: string;
+  };
+}
+
+export type TicketAction =
+  | BuyTicketRequestAction
+  | BuyTicketSuccessAction
+  | BuyTicketFailAction;
```

Comment:
From `Line:9` to `Line:37`, Action type and interfaces should have a separate file.